instrumentation-tests/src/androidTest/java/com/mapbox/navigation/instrumentation_tests/core/EVRerouteTest.kt

```diff
@@ -276,16 +297,144 @@ class EVRerouteTest : BaseTest<EmptyTestActivity>(EmptyTestActivity::class.java)
                 KEY_EV_PRECONDITIONING_TIME to preconditioningTime,
             )
         )
+        checkDoesNotHaveParameters(urlAfterEmptyUpdate, userProvidedCpoiKeys)
+    }
+
+    @Ignore("Unignore when server is ready")
```

Comment:
Yeaaah, well, I prefer to use real server responses in the test (I mean I use mock web server, but predefined responses are taken from the real server), so that we don't test a spherical cow. I usually change manually only something that's very simple and safe.
But I think that with server partial readiness it's already fine to mock the response, I'll do that.